packages/broker/test/integration/plugins/storage/BatchManager.test.ts

```diff
@@ -13,28 +13,23 @@ const keyspace = 'streamr_dev_v2'
 const defaultPublisherId = toEthereumAddress('0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
 const publisherOne = toEthereumAddress('0xbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb')
 
-function buildMsg(
+function buildRecord(
     streamId: string,
-    streamPartition: number,
+    partition: number,
     timestamp: number,
-    sequenceNumber: number,
+    sequenceNo: number,
     publisherId = defaultPublisherId,
     msgChainId = '1',
-    content: any = {}
-) {
-    return new StreamMessage({
-        messageId: new MessageID(
-            toStreamID(streamId),
-            streamPartition,
-            timestamp,
-            sequenceNumber,
-            publisherId,
-            msgChainId
-        ),
-        content: utf8ToBinary(JSON.stringify(content)),
-        signature: hexToBinary('0x1234'),
-        contentType: ContentType.JSON
-    })
+): InsertRecord {
+    return {
+        streamId,
+        partition,
+        timestamp,
+        sequenceNo,
+        publisherId,
```

Comment:
The `publisherId` is always `publisherOne`. Could remove `publisherId` and `msgChainId` parameters from this function and remove `defaultPublisherId` variable.